### PR TITLE
Added useful missing function calls

### PR DESCRIPTION
**Pull request by emptyflash**
I wanted to get y'all's eyes on this.

I implemented these function calls in my fork so I could do some of the things I wanted to do.
Went ahead and documented them in case y'all want to merge for the next version bump.

There's also the matter of the missing context attributes for the stencil buffer,
 i.e. : `canvas.getContext('webgl', {stencil: true})`.
This is a little more important because none of the stencil function calls will work with out it.
I can submit that part as separate PR if that would be preferred.

### CODE REVIEW

**Comment by nphollon** [comment]
Looks mostly OK to me, except why was this line deleted?

`var canvas = document.createElement('canvas');`

**Comment by emptyflash** [comment]
Gasp! I messed up my merge from upstream/master. Fixed.

**Comment by nphollon** [comment]
I am cool with this. Since it's an API change, can we get @johnpmayer to weigh in?

File: src/Native/WebGL.js [hunk 1]
```diff
@@ -533,7 +549,8 @@ var _elm_community$elm_webgl$Native_WebGL = function () {
 
     LOG('Render canvas');
     var canvas = document.createElement('canvas');
-    var gl = canvas.getContext('webgl') || canvas.getContext('experimental-webgl');
+    var attributes = {stencil: true}
```
Comment (Zinggi) [inline src/Native/WebGL.js:552]: I don't think this should be a default setting, many games don't need the stencil buffer.

Better would be to handle this as mentioned here: https://github.com/elm-community/elm-webgl/issues/19

EDIT: now implemented here https://github.com/elm-community/elm-webgl/pull/38

Comment (emptyflash) [reply src/Native/WebGL.js:552]: Good point, I like that solution. Will use once it's merged.

**Comment by w0rm** [comment]
Reopened this pull request to the new repo: https://github.com/elm-community/webgl/pull/5

**Pull request closed without merging**